### PR TITLE
Apply black to GenomeDiagram; update .flake8 &NEWS

### PR DESCRIPTION
**Pull request by peterjc**
This pull request addresses issue #2008, and in particular replaces older pull request #2079.

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -29,6 +29,8 @@ ignore =
     # ===========================================
     E123, # closing bracket does not match indentation of opening bracket's line
           # TODO? (main/Bio/Tests: 3/91/31 occurrences)
+    E203, # gives false positives after running black, see
```
Comment (MarkusPiotrowski) [inline .flake8]: Can you add the error's meaning (whitespace before ':')?

**Comment by peterjc** [comment]
TravisCI is stuck... I will rebase this to refresh it.

**Comment by peterjc** [comment]
While it would be nice to have Leighton's explicit review, he was happy with the earlier version on #2079 so that should be fine.

**Comment by MarkusPiotrowski** [comment]
Travis CI still didn't make it. 
You last commit had something like a comment or command: `[ci skip]`. Could this be related?

**Comment by peterjc** [comment]
Ah - you're probably right Markus.

**Pull request merged**